flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1147,6 +1147,10 @@
     "net.lutris.Lutris": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "net.retrodeck.retrodeck": {
+        "appid-uses-code-hosting-domain": "app-id predates this linter rule",
+        "finish-args-unnecessary-xdg-data-access": "https://github.com/flathub/flathub/issues/4786",
```

Comment:
The current usage is for gtk-3.0 folder, why is that needed? Also, can you please put the actual reason here, not the link.